modules/ROOT/pages/license-influx-update.adoc

```diff
@@ -0,0 +1,40 @@
+= Update InfluxDB License
```

Comment:
Should this be part of the internal docs?
It seems like it has too many internals for the platform.
I'd put this under https://wiki.corp.mulesoft.com/display/SVCS/PCE+3.0+Installation+and+Upgrades.
And leave a short note here to contact Customer Success to get the new license and the instructions to apply it.

Reply:
Good idea. Changed this file to the short note. Moved instructions here: https://wiki.corp.mulesoft.com/display/SVCS/Update+InfluxDB+License